addons/mail/static/src/new/core/thread_service.js

```diff
@@ -12,9 +12,12 @@ import { assignDefined, createLocalId } from "../utils/misc";
 import { Composer } from "../composer/composer_model";
 import { prettifyMessageContent } from "../utils/format";
 import { registry } from "@web/core/registry";
+import { url } from "@web/core/utils/urls";
 
 const FETCH_MSG_LIMIT = 30;
 
+export const DEFAULT_AVATAR = "/mail/static/src/img/smiley/avatar.jpg";
```

Comment:
Misleading export (name) on thread service, as this is the default avatar for a persona, not for a thread.